programming/typescript/challenges/index.js

```
@@ -9,11 +9,40 @@ console.log('Greeting result:', helloAll);
+/**
+ * Challenge 27: Gigasecond
+ * Demonstrates the gigasecond functionality
+*/
+var gigasecond = new Challenges.Gigasecond(new Date('2013-01-07'));
+console.log('Gigasecond date from 2013-01-07:', gigasecond.date());
+var target = new Date('2025-02-28');
+var seconds = Challenges.Gigasecond.secondsToDate(target);
+var days = Challenges.Gigasecond.secondsToDays(seconds); // Returns number of days from now until Dec 31, 2024
+console.log('Days from now until Feb 28, 2025:', days);
+var start = new Date('2013-01-07');
+var gs = new Challenges.Gigasecond(start);
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole module. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)